examples/react-native-cli/android/app/build.gradle

```diff
@@ -116,4 +116,4 @@ dependencies {
     }
 }
 
-apply from: file("../../../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
+apply from: file("../../../../node_modules/.pnpm/@react-native-community+cli-platform-android@12.3.6_encoding@0.1.13/node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
```

Comment:
good good 💚 they really need to solve the monorepo issue with android - ios is so much smoother 🤣